tests/serial/three-solvers/ThreeSolversExplicitImplicit.xml

```diff
@@ -62,8 +62,7 @@
     <participants first="SolverOne" second="SolverThree" />
     <max-time-windows value="10" />
     <time-window-size value="1.0" />
-    <max-iterations value="10" />
-    <min-iteration-convergence-measure min-iterations="3" data="Data2" mesh="MeshA" />
+    <max-iterations value="3" />
```

Comment:
```suggestion
    <min-iterations value="3" />
    <max-iterations value="10" />
```

Reply:
After the conclusion in #800, max=3 is correct.